Makefile

```diff
@@ -1,13 +1,34 @@
-.PHONY: com.docker.slirp.exe install uninstall
+.PHONY: com.docker.slirp.exe com.docker.slirp install uninstall
+
+TARGETS :=
+ifeq ($(OS),Windows_NT)
+	TARGETS += com.docker.slirp.exe
+else
+	TARGETS += com.docker.slirp
+endif
+
+all: $(TARGETS)
 
 EXEDIR=C:\projects\vpnkit
 
 com.docker.slirp:
-	cd src/com.docker.slirp && \
-	oasis setup && \
-	./configure && \
-	make && \
-	cp _build/src/main.native com.docker.slirp
+	$(MAKE) -C src/com.docker.slirp
+
+depends:
+	opam repo list | grep com-docker-slirp || opam repo add com-docker-slirp .
```

Comment:
The PR really does give an easier build and my main issue is the `opam repo add` -- I'll submit a PR with pins instead on top of this branch.  Thanks @djs55 !